ark/utils/spatial_analysis_utils.py

```diff
@@ -17,27 +17,38 @@ def calc_dist_matrix(label_map, path=None):
         path (str):
             path to save file. If None, then will directly return
     Returns:
-        dict:
-            Dictionary containg cells x cells matrices with the euclidian distance between centers
-            of corresponding cells for every fov
-    """
+
+        dist_matrix (dict): contains a cells x cells matrix with the euclidian
+            distance between centers of corresponding cells for every fov,
+            note that each distance matrix is of type xarray"""
+
     # Check that file path exists, if given
+
     if path is not None:
         if not os.path.exists(path):
             raise ValueError("File path not valid")
```

Comment:
while you're refactoring `calc_dist_matrix`, this should be a `FileNotFoundError`